specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/table.json

```diff
@@ -0,0 +1,505 @@
+{
+  "swagger": "2.0",
```

Comment:
As I know, dataplane SDK not support table now. 
When Why we open it in management plan? This will make table expose in all management plan SDK, and management plan rest API.

Reply:
Not absolutely affirmative if data plane SDK exposes CRUD for Tables and Queues, but they do expose REST APIs. With great demand from third-party and internal projects at MSFT, there is a need to surface Dataplane CRUD APIs for table and queue on the Control plane, including SDK support.

Reply:
As per conversation with Chris and peter over the email, we know that we are just surfacing CRUD operation and no query operations. The query API is just a list API underneath. We can move ahead with the review